### DISCUSSION TITLE
Error [BrowserslistError]: Unknown browser query `var updateDb = require('update-browserslist-db')`. Maybe you are using old Browserslist or made typo in query.

### DISCUSSION THREAD

**Discussion by blutuxyz** [tailwindlabs/tailwindcss#12689] [Help]
This happens when I'm trying to build and watch my TailwindCSS files: 
![Screenshot from 2024-01-02 00-01-15](https://github.com/tailwindlabs/tailwindcss/assets/86069069/e23e32bc-477c-4fbf-8c7e-8cddfdc9cac2)
My node version is v17.9.1 (i know its outdated but it also didnt work with 21)
My npm version is v8.11.0

**Comment by wongjn** [comment]
Please consider providing a project that reproduces the undesired behavior, so that we may assist in troubleshooting the issue.

**Reply by blutuxyz** [reply]
Well the problem is that I don't even have a project. I'm just trying to start one and build the files but it doesn't work

Edit: I just found out that it seems like it works on my external hard drive. But then why doesn't it work on my SSD?

**Reply by wongjn** [reply]
You do seem to have a project. By _project_ in this context, I mean a collection of files that you have. You could consider providing this collection of files as a remotely-hosted git repository, if it reproduces the unexpected behavior.
